public/js/p3/widget/app/templates/Homology.html

```diff
@@ -3,7 +3,7 @@
     <div class="appTemplate">
         <div class="appTitle">
             <span class="breadcrumb">Services</span>
-            <h3>BLAST
+            <h3>${applicationLabel}
```

Comment:
This change is correct in terms of using the variable but the current value might not be what we want displayed. Need clarification if the entire front-end and docs, etc will be updated to call it Homology instead of BLAST.

Reply:
I made everything consistent with what is already present in BV-BRC. I tested that the service successfully sent a job to the back end. I think some of us may have talked about it, and BLAST could be the preferred service name because BLAST is well known in the bioinformatics community. 'Homology' is used in the applicationName to send it to the backend and to distinguish it from the old BLAST service.